src/ripple/app/rdb/RelationalDBInterface_nodes.h

```diff
@@ -277,7 +277,11 @@ getTxHistory(
  * @param j Journal.
  * @return Vector of pairs of found transactions and their metadata
  *         sorted in ascending order by account sequence.
- *         Also number of transactions processed.
+ *         Also number of transactions processed or skpiied.
+ *         If this number is >= 0, then it means number of transactions
+ *         processed, if it is < 0, then ~number means number of transactions
+ *         skipped. We need to skip some quantity of transactions if option
```

Comment:
Returning `~number` for the number of transactions skipped is a very strange interface. I'd suggest returning `-number` instead - unless there's some strange reason why `~number` is better, in which case we should document that reason (here and other functions as well).

Reply:
Done